gradebookng/bundle/src/main/bundle/gradebookng.properties

```diff
@@ -389,6 +389,8 @@ grade.log.entry = {0} - Score set to <b>{1}</b> by {2}
 grade.log.none = No grades have been entered for this cell.
 grade.outof = /{0}
 grade.menulabel = Open menu for student {0} and assignment {1} cell
+grade.log.excluded={0} - Score <b>{1}</b> excluded by {2}
```

Comment:
The action the instructor takes is called "Excuse", so let's use this terminology:

`Score of <b>{1}</b> excused by {2}`